src/PexCard.Api.Client/Configure/ConfigurePexApiClientExtensions.cs

```diff
@@ -23,6 +26,8 @@ public static IServiceCollection AddPexApiClient(this IServiceCollection service
                 throw new ArgumentNullException(nameof(configSection));
             }
 
+            RegisterIpResolver(services);
```

Comment:
can't you just move these repeating calls to be inside the one final RegisterPexApiClient method?